Userland/Libraries/LibWeb/Animations/Animation.h

```diff
@@ -106,12 +122,18 @@ class Animation : public DOM::EventTarget {
 
     // https://www.w3.org/TR/web-animations-1/#current-finished-promise
     mutable JS::GCPtr<WebIDL::Promise> m_current_finished_promise;
+    bool m_current_finished_promise_resolved { false }; // FIXME: Can we get this from m_current_finished_promise?
```

Comment:
Is it possible to get a `WebIDL::Promise`'s resolved state? Either way I'll remove this FIXME, just left it here so I remembered it

Reply:
As far as I know it's not stored, and there are many different paths to calling `resolve()`.